lib/p2p/packets/types/SwapFailedPacket.ts

```diff
@@ -44,7 +45,8 @@ class SwapFailedPacket extends Packet<SwapFailedPacketBody> {
       }),
       body: {
```

Comment:
you can wrap this with `removeUndefinedProps` and undo the parser testing workaround, no?

Reply:
Got it, let me try that.